05DatasetAdjustment.ipynb

```diff
@@ -0,0 +1,118 @@
+{
+ "cells": [
```

Comment:
please rename file to something more meaningful like "ReduceFalconDataset.ipynb" or similar.
by the way, not sure how useful the numbers in front of the notebooks name are for you. I only added them to the bootcamp notebooks so participants knew their order. but as you need to keep adding notebooks for your analysis also in between, it would probably make sense to remove these number prefixes.